crates/daphne-server/examples/configuration-helper.toml

```
@@ -21,6 +21,9 @@ base_url = "http://127.0.0.1:8788"
 allow_taskprov = true
 default_num_agg_span_shards = 4
 
+[service.peer_auth.helper]
+leader_token = "I-am-the-leader" # SECRET
```

This doesn't have any risk of being deployed.
Likewise below.

```suggestion
leader_token = "I-am-the-leader"
```

---

I thought this comment served as a kind of documentation to show that this value is supposed to be a secret in a production environment

---

That sounds like a reasonable interpretation. In that case, this comment should also be added below:
```
[service.leader_to_helper_tokens]
helper = "I-am-the-leader"
```